packages/core/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "outDir": "dist",
     // "noImplicitAny": true,
     "paths": {
-      "@core/*": ["./*"]
+      "@core/*": ["./*"],
+      "@viz/*": ["../../viz/src/*"]
```

Comment:
This was added because of [this error](https://github.com/lumada-design/hv-uikit-react/actions/runs/6562065132/job/17823583163). For some reason, `typecheck` was running locally but it was breaking in the workflow. 
We can try to remove this and see if it's break again because it didn't the first time around. 

But we have this kind of config in other packages when a package is importing from another one. For example here, the `viz` package needs to config the `core` aliases because otherwise it wouldn't build as `viz` doesn't know how to resolve the aliases.

```
"paths": {
      "@core/*": ["../../core/src/*"],
      "@viz/*": ["./*"]
}
```

It would be nice to have a better solution but adding the alias in the `tsconfig` was the solution we found at the time to make all of this work.

Reply:
All right, thanks for the explanation 👍 

It would've been even nicer if we didn't have a `@viz/*` alias 😜